src/Numerics/Optimizers/ExhaustiveOptimizer/CMakeLists.txt

```diff
@@ -6,21 +6,31 @@ find_package(ITK REQUIRED)
 include(${ITK_USE_FILE})
 
 
-add_executable(ExhaustiveOptimizer Code.cxx)
+add_executable(ExhaustiveOptimizer Code.cxx exhaustiveoptimizer.py)
 target_link_libraries(ExhaustiveOptimizer ${ITK_LIBRARIES})
 
 install(TARGETS ExhaustiveOptimizer
   DESTINATION bin/ITKExamples/Numerics/Optimizers
   COMPONENT Runtime
   )
 
-install(FILES Code.cxx CMakeLists.txt
+install(FILES Code.cxx exhaustiveoptimizer.py CMakeLists.txt
   DESTINATION share/ITKExamples/Code/Numerics/Optimizers/ExhaustiveOptimizer/
   COMPONENT Code
   )
 
 
 enable_testing()
-add_test(NAME ExhaustiveOptimizerTest
-  COMMAND ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/ExhaustiveOptimizer)
 
+set(fixed_image ${CMAKE_CURRENT_BINARY_DIR}/orange.jpg)
+set(moving_image ${CMAKE_CURRENT_BINARY_DIR}/apple.jpg)
+
+add_test(NAME ExhaustiveOptimizerTest
+  COMMAND ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/ExhaustiveOptimizer
+	${fixed_image}
```

Comment:
Spaces are preferred over tabs. Please invoke clang-format (Format Document in VS).

Reply:
I've gone back and corrected tabs manually, but it doesn't seem that the Format Document command is available for CMakeLists.txt files in my VS editor. I can format cxx and Python documents without issue. Is there a fix for this?

Reply:
You are right, Format Document should only work for C++ (and Python).